src/Numerics/DGMethods/FVModel.jl

```diff
@@ -0,0 +1,483 @@
+using StaticArrays
```

Comment:
Let's rename this file, and put it in a new folder: `src/Numerics/FVMethods/FVMethods.jl`